addons/l10n_it_edi/tests/test_account_move_send.py

```diff
@@ -78,3 +79,47 @@ def _get_default_extra_edis(self, move):
         self.assertTrue(invoice2.invoice_pdf_report_id)
         self.assertTrue(invoice2.l10n_it_edi_attachment_id)
         self.assertFalse(invoice2.is_being_sent)
+
+    def test_invoice_two_downpayments_with_l10n_it_edi_xml_export(self):
```

Comment:
This test file is just the place to test the interaction with the wizard and its flags.
All the tests for the values from the export of the EDI should stay in `test_edi_export.py`.

You can make a test method that just confirm the order, post the invoices, then use `_assert_export_invoice` with a sample XML, like all the other tests do, without involving the wizard, which is not the object of the test and often changes (more than export).

Reply:
thanks, I understand why there was not that much test in this file then 👍 

I made the modifications as well as the change in the code (the `negative_downpayment_lines` didn't seem useful)

Reply:
> (the negative_downpayment_lines didn't seem useful)

Sorry, I briefly used something new without looking what `downpayment_lines` was, gj 👍🏻 

> edit: I only removed account.move.send wizard

That's fine.